py/makeqstrdefs.py

```diff
@@ -26,14 +25,15 @@ def write_out(fname, output):
         with open(args.output_dir + "/" + fname + ".qstr", "w") as f:
             f.write("\n".join(output) + "\n")
 
+
 def process_file(f):
     output = []
     last_fname = None
-    outf = None
     for line in f:
         if line and line[0:2] == "# ":
-            comp = line.split()
-            fname = comp[2]
+            # There can be spaces in the filename - e.g. "<command line>" so we use regular expression to parse it:
+            match = re.match('#\s+\d+\s+(?P<fname>"[^"]*").*', line)
```

Comment:
I have removed the assert for quotes outside of the regex, modified the regex to match just the file name without quotes and added assert to check that there is a match.